### ISSUE TITLE
Redesign "Library" tab to be more consistent with UI

### ISSUE THREAD

**Issue by Sadwhy** [Sangwan5688/BlackHole#676]
In the latest update the settings page was revamped to look really good with a few minor animations. But the problem is that it's now inconsistent with the "Library" tabs UI. It's bland. The UI elements are a little small. There's too much empty space. Overall not a good look. I suggest that you redesign this tab to look bigger, rounder with more functionalities and smooth animations. Like the playlist bar showing frequently played playlists underneath the bar in a square shape. And more I can't mention.

**Comment by abhishek09827** [comment]
Hello @Sadwhy pls assign this task to me

**Comment by Sadwhy** [comment]
> Hello @Sadwhy pls assign this task to me
> 

Yeah I would but I'm too unfamiliar with GitHub to know what to press.
